src/main/java/blackjack/view/InputView.java

```diff
@@ -33,16 +36,32 @@
                 .collect(Collectors.toList());
     }
 
-    public static Boolean inputDrawCardAnswer(final Player user) {
+    public static boolean inputDrawCardAnswer(final Player user) {
+        System.out.println();
         System.out.println(user.getName() + DRAW_CARD_MESSAGE);
         String answer = scanner.nextLine();
         return validateDrawCardAnswer(answer);
     }
 
-    private static Boolean validateDrawCardAnswer(final String answer) {
+    private static boolean validateDrawCardAnswer(final String answer) {
         if (!answer.equals(YES) && !answer.equals(NO)) {
```

Comment:
```suggestion
        if (answer.equals(YES) || answer.equals(NO)) {
```

A도 아니고 B도 아니면 ~이다.
A이거나 B이면 ~가 아니다.

가 더 자연스럽지 않을까요 ?

* 참고 - [드모르간의 법칙](https://m.blog.naver.com/PostView.naver?isHttpsRedirect=true&blogId=asd7979&logNo=30106982953)